### ISSUE TITLE
Why is an extra clef displayed here?

### ISSUE THREAD

**Issue by fablau** [opensheetmusicdisplay/opensheetmusicdisplay#1461]
Hello again,
Here is another strange issue that I came up with in this Mozart's sonata. Look at the shot below:

![ExtraClef](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/10005316/e63f2577-29d2-41a5-96a3-309f653b9580)

For some reason, on bar 29, left hand, OSMD has added an extra F clef, which is not defined inside the XML file. I have circled that clef in red. Why's that?

I am attaching the MusicXML file of it. I am eager to know your thoughts about this as well.

Thanks again!
[01AllegroOSMDtest.musicxml.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/12777698/01AllegroOSMDtest.musicxml.zip)

**Comment by fablau** [comment]
Yes Simon, I didn't export just those two bars because the problem didn't appear, that's why ;)

Ok, so I guess I'll have to user MUseScore to fix it. Just a question: have you idea what are the "conditions" to create such a problem, as far as you know? I am working on a system to optimize the MusicXML file before feeding it to OSMD, so any more info is very welcome. Thanks again :)

**Comment by sschmidTU** [comment]
Ah, yes, well, it can happen that some issues only appear under very certain conditions in larger scores. Unfortunately I'm not sure what the exact conditions were in this case.
I already fixed the issue, so it's not going to happen anymore. But for future issues, it would of course be nice to know them.

**Comment by fablau** [comment]
Fantastic, thank you Simon! You are my savior! ;)
